src/app/lib/utilities/fetchCircuitBreaker/index.js

```diff
@@ -0,0 +1,25 @@
+import 'isomorphic-fetch';
```

Comment:
Wow would we be integrating this with Simorgh and our current method of data fetching?

Reply:
I believe that's the plan. However, since it's a drop-in replacement for `fetch` we can integrate it incrementally.

Reply:
Sorry, I meant that i terms of this circuit breaker in general, Or is this already working/integrated in this PR